.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  pull_request: {}
+  push: {}
+
+name: Continuous Integration
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [12, 14, 16, 18]
```

Comment:
I started with 12 since that’s what `datadog/datadog-api-client` currently supports, and we don’t have a clear sense of minimum requirements otherwise (https://github.com/dbader/node-datadog-metrics/issues/67#issuecomment-1250475577)